packages/venia-ui/lib/queries/getProductDetailByName.graphql

```diff
@@ -1,5 +1,5 @@
 query productDetailByName($name: String, $onServer: Boolean!) {
-    products(filter: { name: { eq: $name } }) {
+    products(filter: { name: { match: $name } }) {
```

Comment:
We need to talk about what this means. The `filter.name` type changed from "equals" type in 2.3.3 to "match" type in 2.3.4. In order to support both versions we would have to somehow switch the variable prop dynamically based on the backend _or_ GraphQL team will have to provide some method which works on both versions.

cc @nishantjariwala @awilcoxa @cherdman

Reply:
@cpartica